gitea/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       PROXY_AUTH_ADD: "false"
 
   server:
-    image: gitea/gitea:1.16.8-rootless@sha256:8a6ad22694c76122ae075af1153a544577724c45cb45af44963aa78dc179cc46
+    image: gitea/gitea:1.17.3
```

Comment:
Yes exactly, we need to use a specific tag for a specific version of the app. We're just missing the Docker image digest. You can get this by running: `docker pull <image>` and it will display at the end.